smartrouter/image.yaml

```diff
@@ -93,6 +93,9 @@ modules:
           - name: dynamic-resources
           - name: rhpam-7-smartrouter
           - name: os-logging
+          - name: os-eap-python
```

Comment:
@errantepiphany isn't better just install the python package instead use a cekit module?

Reply:
Is this python module that is required the JBOSS_HOME?

Reply:
@spolti I used this variable in the new module "jboss-kie-common". In the "jboss-kie-smartrouter" module this var does not exist so I had to define it.

Reply:
@spolti The only problem is that if we just install the python package without installing via this eap modules, we'd be missing this part:
https://github.com/jboss-container-images/jboss-eap-modules/blob/7.2.x-openjdk11/os-eap-python/3.6/configure.sh#L4

@msmagnanijr Your usage of os-eap-python module here won't work without also adding jboss-eap-modules to image.yaml, branch-overrides.yaml, and tag-overrides.yaml .

Both: If we don't want to add the above dependencies on jboss-eap-modules (os-eap-python, specifically), then we could add a jboss-kie-python module to jboss-kie-modules... I don't know if it's worth it, though... Thoughts?

Reply:
I think creating a new module is easy but will it really if we are going to use this module enough to justify creating a new one? For the sake of modularization (decoupling) I think it interesting.

Reply:
@errantepiphany AFAIK the alternatives is used to use a specific binary version in case you have more than one python installation, the same way we do for java.

do you know what is the python version that comes with ubi8 images?

I we decide to move on the module direction I would choose to use the one provided by eap.

Reply:
@luck3y , the os-eap-python 3.6 module not only installs that version of python, but also sets the alternatives to point to it in the configure.sh script. Was this step necessary for some reason? It would seem only necessary if there was more than one version of python installed, per @spolti 's comment above.

Reply:
@errantepiphany RHEL 8 has system-python set to python3, but you can't use it until you install the python36 user modules (it ends up making a virtualenv I think).

As I recall the alternatives bit was because even after installing the python36 RPMs /etc/alternatives/python was still set to 'nopython'. This may have been a bug in an earlier UBI content set, so perhaps it isn't needed, but it doesn't do any harm. 

The existence of the module is basically because of this and the very specific versioned package name 'python36', at some point there will also be a 'python37' and they'll have to coexist for various images.

Reply:
@luck3y Thank you for the explanation. @spolti I'm going to merge this and we can revisit the dependency of smartrouter on an eap module in the future, if needed.